### PR TITLE
Mention MariaDB as well

### PR DESCRIPTION
**Pull request by iangilfillan**

### CODE REVIEW

File: session/database.rst [hunk 1]
```diff
@@ -390,8 +390,8 @@ file and run the migration with the following command:
 
     $ php bin/console doctrine:migrations:migrate
 
-MySQL
-.....
+MariaDB/MySQL
+.............
```
Comment (OskarStark) [inline session/database.rst:383]:
```suggestion
.. _mysql:

MariaDB/MySQL
.............
```

File: session/database.rst [hunk 2]
```diff
@@ -166,11 +166,11 @@ parallel and only the first one stored the CSRF token in the session.
     If you use Memcached instead of Redis, follow a similar approach but replace
     ``RedisSessionHandler`` by :class:`Symfony\\Component\\HttpFoundation\\Session\\Storage\\Handler\\MemcachedSessionHandler`.
 
-Store Sessions in a Relational Database (MySQL, PostgreSQL)
+Store Sessions in a Relational Database (MariaDB, MySQL, PostgreSQL)
 -----------------------------------------------------------
```
Comment (OskarStark) [inline session/database.rst:170]:
```suggestion
Store Sessions in a Relational Database (MariaDB, MySQL, PostgreSQL)
--------------------------------------------------------------------
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
2 minor comments

**Comment by javiereguiluz** [comment]
Ian thanks and congrats on your first Symfony Docs contribution! We fixed some minor syntax issues while merging but your changes were good!

File: session/database.rst (hunk 1 above)
Comment (javiereguiluz) [reply session/database.rst:383]: Done while merging.

**Pull request merged**